### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by dnol777**
Forgot to make this pull request! Just added a text area to contact form so that questions can be asked. Once Mr. Miles let us know how he wants questions to be directed, we can change it up as necessary.

### CODE REVIEW

File: modules/core/client/views/lightbox/contact.client.view.html [hunk 1]
```diff
@@ -24,6 +24,17 @@
 
         </div>
 
+        <div class="row">
+
+              <div class="input-field col s12">
+
+                  <textarea id="textarea1" class="materialize-textarea"></textarea>
+                  <label for="textarea1">What would you like know?</label>
```
Comment (ssk13) [inline modules/core/client/views/lightbox/contact.client.view.html]: *"What would you like to know?"

Comment (dnol777) [inline modules/core/client/views/lightbox/contact.client.view.html]: Added the quotations marks!

Comment (ssk13) [inline modules/core/client/views/lightbox/contact.client.view.html]: @dnol777 
ahhh no I wasn't saying to add quotes - there should be a 'to' in between 'like' and 'know'...

Comment (dnol777) [inline modules/core/client/views/lightbox/contact.client.view.html]: Lol changed it. This is what a lack of sleep does to you....

**Pull request merged**